container_registry/container_analysis/src/sample/quickstart.go

```diff
@@ -0,0 +1,149 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Sample quickstart for getting vulnerabilities from the Container Analysis API: https://cloud.google.com/container-registry/docs/vulnerability-scan-go
+package sample
+
+// [START containeranalysis_imports_quickstart]
+
+import (
+	"context"
+	"fmt"
+	wait "k8s.io/apimachinery/pkg/util/wait"
+	"log"
+	"time"
+
+	containeranalysis "cloud.google.com/go/containeranalysis/apiv1beta1"
+	"google.golang.org/api/iterator"
+	discovery "google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/discovery"
+	grafeaspb "google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/grafeas"
+	vulnerability "google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/vulnerability"
+)
+
+// [END containeranalysis_imports_quickstart]
+
+// [START containeranalysis_poll_discovery_occurrence_finished]
+
+// pollDiscoveryOccurrenceFinished returns the discovery occurrence for a resource once it reaches a finished state.
+func pollDiscoveryOccurrenceFinished(resourceUrl, projectID string, timeout time.Duration) (*grafeaspb.Occurrence, error) {
```

Comment:
I haven't been a part of the discussions around imports in samples, but wouldn't this cause issues for some other languages? I think it would seem weird to have an "imports" region tag for some languages, and include imports in the samples for others

Reply:
The goal is to have idiomatic samples for each language. So, each one is different. I'm not sure where the other languages landed with imports in regions. But, most docs don't have an imports section at the top of the page, so it makes sense to have them with the sample.

If this is the current setup, find by me to keep it as-is.